scripts/docker-build.sh

```diff
@@ -54,11 +54,11 @@ build_server() {
   rm -rf packages/*/coverage || true
   rm -rf packages/*/config/{local,development,test}.* || true
 
-  remove_irrelevant_packages "$package"
-
   # build the world
   npm run build
 
+  remove_irrelevant_packages "$package"
+
```

Comment:
@passcod this is the simplest way to ensure all the translations are imported. Just build everything first before removing irrelevant packages. It's a bit wasteful though obviously as we build the irrelevant packages, so I can put some time into reworking this script to just build the relevant packages if you like?

Could also split out the scrap translations from the build process. One thing that I'm a little worried about is if we ever rebuild the `@tamanu/upgrade` package later on without realising we'll replace the default-translations.json with whatever's left here